src/controllers/controller.line.js

```diff
@@ -327,6 +327,7 @@ module.exports = function(Chart) {
 			var index = element._index;
 			var custom = element.custom || {};
 			var model = element._model;
+			var pointOpts = this.chart.options.elements.point; 
```

Comment:
there's a trailing space on this line that's causing the build to fail

Reply:
this PR looks good to me after the trailing space is fixed

Reply:
@benmccann I have removed the extra space and the build is successful